htdocs/societe/checkvat/checkVatGr.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/* Copyright (C) 2016       Spiros Ioannou
+ * Copyright (C) 2017       Marios Kaintatzis
+ * Copyright (C) 2023       Nick Fragoulis
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ *  \file       htdocs/societe/checkvat/checkVatGr.php
+ *  \ingroup    societe
+ *  \brief      Request VAT details from the Greek Ministry of Finance GSIS SOAP web service
+ */
+
+require "../../main.inc.php";
+
+$username = getDolGlobalString('AADE_WEBSERVICE_USER'); // Get username from request
+$password = getDolGlobalString('AADE_WEBSERVICE_KEY'); // Get password from request
+$myafm = getDolGlobalString('MAIN_INFO_TVAINTRA'); // Get Vat from request
+$afm = (isset($_REQUEST['afm']) ? $_REQUEST['afm'] : ''); // Get client Vat from request
```

Comment:
Do not use $_REQUEST
Use instead
$afm = GETPOST('afm');

Reply:
fixed